marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs

```diff
@@ -300,14 +300,20 @@ function validatePropertyMapping(fullMapping, propertyName, sourcedFrom) {
     // As of trunk 10.0-20190916, mappings are being validated against entity schemas in the schema database.
     // This doesn't seem expected, as the validation will almost always fail.
     // Thus, this is not using es.mappingCompile, which does validation, and just invokes the transform instead.
-    let stylesheet = xdmp.xsltInvoke("/MarkLogic/entity-services/mapping-compile.xsl", xmlMapping);
-    xdmp.xsltEval(stylesheet, [], {staticCheck: true});
+    let stylesheet = fn.head(xdmp.xsltInvoke("/MarkLogic/entity-services/mapping-compile.xsl", xmlMapping));
+    xdmp.xsltEval(removeStandardFunction(stylesheet), [], {staticCheck: true});
   } catch (e) {
     // TODO Move this into a separate function for easier testing?
     return getErrorMessage(e);
   }
 }
 
+//This function removes import of "standard-library.xqy" from mapping xslt. Server Bug 54051
```

Comment:
Nice, I was just going to say that we probably need a comment here to explain what's going on.